From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Thu, 17 Mar 2011 13:34:53 +0100
Subject: [PATCH] teach ncurses-config about sysroot

This patch teaches ncurses-config about sysroot

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
[ported from ncurses 6.1 to 6.2]
Signed-off-by: Robert Schwebel <rsc@pengutronix.de>
---
 misc/ncurses-config.in |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

--- ncurses-6.2/misc/ncurses-config.in	2020-02-03 00:34:34.000000000 +0100
+++ ncurses-6.2-ptx/misc/ncurses-config.in	2020-07-25 23:28:48.123350040 +0200
@@ -149,10 +149,10 @@
 	--cflags)
 		INCS="@PKG_CFLAGS@"
 		if [ "x@WITH_OVERWRITE@" = xno ]; then
-			INCS="$INCS -I${includesubdir}"
+			INCS="$INCS -I${SYSROOT}${includesubdir}"
 		fi
 		if [ "${includedir}" != /usr/include ]; then
-			INCS="$INCS -I${includedir}"
+			INCS="$INCS -I${SYSROOT}${includedir}"
 		fi
 		sed -e 's,^[ ]*,,' -e 's, [ ]*, ,g' -e 's,[ ]*$,,' <<-ENDECHO
 			$INCS
@@ -235,7 +235,7 @@
 		echo $INCS
 		;;
 	--libdir)
-		echo "${libdir}"
+		echo "${SYSROOT}${libdir}"
 		;;
 	--mandir)
 		echo "${mandir}"
